src/Raven.Studio/typescript/common/bindingHelpers/datePickerBindingHandler.ts

```diff
@@ -35,13 +35,24 @@ class datePickerBindingHandler {
 
             const newDate = moment(ev.date);
             const value = valueAccessor();
+            
             value(newDate);
         });
     }
 
     // Called by Knockout each time the dependent observable value changes.
     update(element: HTMLElement, valueAccessor: any, allBindings: KnockoutAllBindingsAccessor, viewModel: any, bindingContext: any) {
-        const date : moment.Moment = ko.unwrap(valueAccessor());
+        
+        const dateToUse = allBindings().datepickerDateToUse && allBindings().datepickerDateToUse() || null;
```

Comment:
why we need 2 properties to control default date? Can't we use one?

Reply:
fixed to:
`const dateToUse = allBindings().datepickerDateToUse() || null;`

Reply:
why we need datepickerDateToUse at all?

Reply:
Can't you use valueAccessor?

Reply:
done - using `valueAccessor `now